pages/account/manage/event/[[...data]].js

```diff
@@ -57,6 +57,7 @@ export default function ManageEvent({ BASE_URL, event }) {
   const [endDate, setEndDate] = useState("");
   const [speakingTopic, setspeakingTopic] = useState(event.speakingTopic || "");
   const [tags, setTags] = useState(event.tags || []);
+  const inputRef = useRef(null);
```

Comment:
I'm not sure why this variable named `inputRef`. I think naming the variable simply `inputRef` might not be very descriptive, because I see in the scope of this function has multiple input fields.

Reply:
Yep 👍  Inside the tag component it can be `inputRef` but outside it should be `tagInputRef`

Reply:
Yeah that makes sense

Reply:
sorry for the late reply, I made the changes as you requested and pushed the changes.